cloudfour/cloudfour.com-patterns
# Netlify config

## Overview

This resolves #485 by adding two files to the repository:

- A [Netlify configuration file](https://docs.netlify.com/configure-builds/file-based-configuration/) specifying build steps for this branch.
- A `.nvmrc` file specifying a Node version of `lts/*`. Without this, Netlify seems to default to the earliest Node version it supports (`4.x`), which causes the build to break.

## Testing

1. Try [the deploy preview](https://5e500d2fa952a900089ec90b--cloudfour-patterns.netlify.com/).
1. Observe that static checks on this branch pass.
